code/android-core-library/src/test/java/com/adobe/marketing/mobile/launch/rulesengine/LaunchRulesEvaluatorTests.kt

```diff
@@ -70,8 +72,8 @@ class LaunchRulesEvaluatorTests {
         launchRulesEvaluator.replaceRules(listOf())
         verify(extensionApi, Mockito.times(1)).dispatch(eventCaptor.capture())
         assertNotNull(eventCaptor.value)
-        assertEquals("com.adobe.eventtype.rulesengine", eventCaptor.value.type)
-        assertEquals("com.adobe.eventsource.requestreset", eventCaptor.value.source)
+        assertEquals(EventType.RULES_ENGINE, eventCaptor.value.type)
```

Comment:
Can we use local String in tests so it fails in case of an accidental change to the constants class? cc: @yangyansong-adbe

Reply:
As discussed let us reference the constants from EventType/EventSource classes